### PR TITLE
MM-40165 - remaining ux improvements for licensing page

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR takes care of the style enhancements left as part of the PR 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-40165

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/9372#issuecomment-970589384

#### Screenshots
Captura de pantalla 2021-11-17 a las 20 25 07

Captura de pantalla 2021-11-17 a las 20 38 13

Captura de pantalla 2021-11-17 a las 20 25 19

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx [hunk 1]
```diff
@@ -57,6 +57,12 @@ const EnterpriseEditionLeftPanel: React.FC<EnterpriseEditionProps> = ({
 
     return (
         <div className='EnterpriseEditionLeftPanel'>
+            {isTrialLicense && <div className='pre-title'>
+                <FormattedMessage
+                    id='admin.license.enterpriseEdition'
+                    defaultMessage='Enterprise Edition'
+                />
+            </div>}
```
Comment (marianunez) [inline components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx:65]: Why is this title only for trial licenses?

**Review by marianunez — APPROVED** [review]
LGTM, thanks @pablovelezvidal!

File: components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx (hunk 1 above)
Comment (pablovelezvidal) [reply components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx:65]: According to the design it only appears in Start and when it is a Trial license. @annelieseklein can confirm that.

**Review by AGMETEOR — APPROVED** [review]

File: components/admin_console/license_settings/license_settings.tsx [hunk 2]
```diff
@@ -415,8 +415,9 @@ export default class LicenseSettings extends React.PureComponent<Props, State> {
                 <div className='admin-console__wrapper'>
                     <div className='admin-console__content'>
                         <div className='admin-console__banner_section'>
-                            {(license.IsLicensed !== 'true') && (this.props.prevTrialLicense?.IsLicensed !== 'true') &&
-                                free30DayTrial(
+                            {this.props.enterpriseReady && (license.IsLicensed !== 'true') &&
```
Comment (pablovelezvidal) [inline components/admin_console/license_settings/license_settings.tsx:418]: Adding the enterpriseReady condition we address ticket https://mattermost.atlassian.net/browse/MM-40169

Comment (marianunez) [reply components/admin_console/license_settings/license_settings.tsx:418]: Awesome, thanks for incorporating this fix here 👍

**Comment by marianunez** [comment]
@stevemudie I've went ahead and marked this as QA Deferred as these are small clean up. This can be tested during RC testing as part of the feature testing.

**Comment by annelieseklein** [comment]
@pablovelezvidal the button heights on the right hand card still look a bit short compared to the component size

**Comment by annelieseklein** [comment]
Screen Shot 2021-11-18 at 11 45 01 AM
Can we add the "Enterprise Edition" subtitle above the "Mattermost Enterprise" title?
The font at the bottom of the license details in "This software...." looks like it is Helvetica when it should be Open Sans

**Comment by annelieseklein** [comment]
Button heights should be 48px

**Review by annelieseklein — APPROVED** [review]

**Pull request merged**